siocli/lib/SqlDump.cpp

```diff
@@ -36,43 +36,111 @@ namespace {
 
 using namespace siodb::iomgr::dbengine;
 
+/**
+ * Database information
+ */
 struct DatabaseInfo {
+    /** Initializes structure DatabaseInfo. */
+    DatabaseInfo() = default;
+
+    /** Database name */
     std::string m_name;
+
+    /** Cipher ID of the database */
     std::string m_cipherId;
+
+    /** Cipher key of the database */
     BinaryValue m_cipherKey;
 };
 
+/**
+ * Column constraint information
+ */
 struct ColumnConstraint {
+    /** Initializes structure ColumnConstraint. */
+    ColumnConstraint() = default;
```

Comment:
1. noexcept
2. Must explicitly initialize members of fundamental types and enums (in this case `m_type` and `m_constraintDefinitionId`

Reply:
BinaryValue default constructor is not noexcept

Reply:
Please see my last 2 comments. Make default noexcept constructor in the `stdext::buffer`